invui-core/pom.xml

```diff
@@ -19,6 +19,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>dev.folia</groupId>
+            <artifactId>folia-api</artifactId>
+            <version>1.20.1-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Not a fan of the duplicate api dependencies. Also as far as I know folia-api is not different from paper-api.